app/knight_stats.py

```diff
@@ -0,0 +1,33 @@
+class Knight:
+    def __init__(self, equipment: dict) -> None:
```

Comment:
maybe "name", "hp", "power" is not "equipment"

Reply:
I agree 👍 Implicit expectiations, when you create Knight object, you don't know what exactly should be in equipment. And is `name` and equipmet?
To understand that I will need to go in this module and read the class logic. Change that